beetsplug/web/__init__.py

```diff
@@ -337,6 +338,11 @@ def func(lib, opts, args):
                     r"/*": {"origins": self.config['cors'].get(str)}
                 }
                 CORS(app)
+
+            if self.config['subsonic']:
+                from .subsonic import subsonic_routes
+                app.register_blueprint(subsonic_routes)
+
```

Comment:
It seems like this strategy could potentially be used to implement the extensible web plugin, #718!

Imagine implementing this as a separate plugin called `subsonic`. In that plugin's `__init__.py`, you could do something like this:

```py
# beets/beetsplug/subsonic/__init__.py
from beets.beetsplug.web import register_blueprint

subsonic_routes = Blueprint('subsonic', 'subsonic')

@subsonic_routes.route('/rest/getMusicFolders.view', methods=['GET', 'POST'])
def v_music_folders():
    pass # whatever

register_blueprint(subsonic_routes)
```

On the `web/__init__.py` side, it would work like this:

```py
# beets/beetsplug/web/__init__.py

_BLUEPRINTS = []
def register_blueprint(blueprint):
    _BLUEPRINTS.append(blueprint)

class WebPlugin(BeetsPlugin):
    # ...
    def commands(self):
        # ...
        def func(lib, opts, args):
            # ...
            for blueprint in _BLUEPRINTS:
                app.register_blueprint(blueprint)
```

This would require almost no changes to your patch, except that you would have to enable subsonic as an independent plugin. But when you run `beet web`, it will (I think?) include the subsonic routes.

@sampsyo what do you think? Am I wrong about how plugin imports and bootstrapping works?